packages/ui/package.json

```diff
@@ -39,6 +39,7 @@
     "@tokens-studio/graph-engine-nodes-audio": "*",
     "@tokens-studio/graph-engine-nodes-design-tokens": "*",
     "@tokens-studio/graph-engine-nodes-image": "*",
+    "@tokens-studio/icons": "0.1.3",
```

Comment:
update that to 0.1.4 to resolve some of the layout issues the icons had

Reply:
updated